### ISSUE TITLE
Refactor DateTime and TimeSpan humanization to use ToQuantityExtensions.ToQuantity to quantify time units 

### ISSUE THREAD

**Issue by MehdiK** [Humanizr/Humanizer#205]
Brought over from #192

**Comment by alimeziane** [comment]
Hi,
I am actually working on this and I would like to know whether the goal is to get rid of the functions time span and date time and replace them with quantity or to keep them all ?
Thanks.

**Comment by alimeziane** [comment]
Hi,
I am actually working on this and  I have doubts about the need. This is what I understood: the aim is to restructure humanization of Datetime to have the same parameters as there in ToQuantity( like ShowQuantityAs.Words). 
Is that correct?
Thanks

**Comment by hazzik** [comment]
Covered by #405

**Comment by hazzik** [comment]
@alimeziane not exactly. The idea was to use Inflector for words as "second", "minute", "hour", etc.

**Comment by alimeziane** [comment]
@hazzik , In the PR #405 , i added the parameter "ShowQuantityAs" to TimeSpan and DateTime humanization , it will select the display mode (numbers or words), for example :
TimeSpan.FromDays(15).Humanize(2,showQuantityAs : ShowQuantityAs.Words) --> two weeks , one day
DateTime.Now.AddDays(12).Humanize(false,showQuantityAs : ShowQuantityAs.Words) --> twelve days from now

So I think it solves the #290 issue , but I do not see anything added to solve the current issue.
If you have an example that would be cool.

**Comment by hazzik** [comment]
>So I think it solves the #290 issue , but I do not see anything added to solve the current issue.
If you have an example that would be cool.

You're right.

This issue is about internal implementation of time fraction quantification, to remove need of resources

**Comment by SimonCropp** [comment]
@MehdiK  still think this is worthwhile?

**Comment by clairernovotny** [comment]
Thank you for the original design discussion. Humanizer’s localization and time-unit formatting architecture has since been replaced by the v3 generated locale/formatter pipeline, so this specific internal refactor is no longer applicable. I am closing it as superseded by those structural changes.
